src/tiled/tilesetdock.cpp

```diff
@@ -392,8 +390,12 @@ void TilesetDock::dropEvent(QDropEvent *e)
 void TilesetDock::currentTilesetChanged()
 {
     if (const TilesetView *view = currentTilesetView())
+    {
```

Comment:
Coding style: please place the `{` on the same line as the `if` (like for `for` and `while`, except for functions).